
-----
Test: tests/test_scenario_1.py::test_scenario_1 (call)
Start time: 2024-11-06 22:44:07.188117
Start name method: Scenario 1 start


-----
Error occurred at: 2024-11-06 22:44:18.202011
Error: Message: unknown error: cannot determine loading status
from target frame detached
  (Session info: chrome=130.0.6723.92)
Stacktrace:
	GetHandleVerifier [0x00007FF6C3023B05+28005]
	(No symbol) [0x00007FF6C2F88400]
	(No symbol) [0x00007FF6C2E25639]
	(No symbol) [0x00007FF6C2E1387B]
	(No symbol) [0x00007FF6C2E1151E]
	(No symbol) [0x00007FF6C2E11FCF]
	(No symbol) [0x00007FF6C2E21157]
	(No symbol) [0x00007FF6C2E3385D]
	(No symbol) [0x00007FF6C2E391FA]
	(No symbol) [0x00007FF6C2E12734]
	(No symbol) [0x00007FF6C2E333DF]
	(No symbol) [0x00007FF6C2EBB795]
	(No symbol) [0x00007FF6C2E9B7E3]
	(No symbol) [0x00007FF6C2E675C8]
	(No symbol) [0x00007FF6C2E68731]
	GetHandleVerifier [0x00007FF6C331647D+3118813]
	GetHandleVerifier [0x00007FF6C3366CD0+3448624]
	GetHandleVerifier [0x00007FF6C335CF4D+3408301]
	GetHandleVerifier [0x00007FF6C30EA45B+841403]
	(No symbol) [0x00007FF6C2F9345F]
	(No symbol) [0x00007FF6C2F8F4D4]
	(No symbol) [0x00007FF6C2F8F66D]
	(No symbol) [0x00007FF6C2F7EBC9]
	BaseThreadInitThunk [0x00007FFBF3FC259D+29]
	RtlUserThreadStart [0x00007FFBF5E4AF38+40]

Traceback: Traceback (most recent call last):
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\base\base_class.py", line 89, in assert_url
    get_url = self.driver.current_url
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\.venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 470, in current_url
    return self.execute(Command.GET_CURRENT_URL)["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\.venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 380, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\.venv\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status
from target frame detached
  (Session info: chrome=130.0.6723.92)
Stacktrace:
	GetHandleVerifier [0x00007FF6C3023B05+28005]
	(No symbol) [0x00007FF6C2F88400]
	(No symbol) [0x00007FF6C2E25639]
	(No symbol) [0x00007FF6C2E1387B]
	(No symbol) [0x00007FF6C2E1151E]
	(No symbol) [0x00007FF6C2E11FCF]
	(No symbol) [0x00007FF6C2E21157]
	(No symbol) [0x00007FF6C2E3385D]
	(No symbol) [0x00007FF6C2E391FA]
	(No symbol) [0x00007FF6C2E12734]
	(No symbol) [0x00007FF6C2E333DF]
	(No symbol) [0x00007FF6C2EBB795]
	(No symbol) [0x00007FF6C2E9B7E3]
	(No symbol) [0x00007FF6C2E675C8]
	(No symbol) [0x00007FF6C2E68731]
	GetHandleVerifier [0x00007FF6C331647D+3118813]
	GetHandleVerifier [0x00007FF6C3366CD0+3448624]
	GetHandleVerifier [0x00007FF6C335CF4D+3408301]
	GetHandleVerifier [0x00007FF6C30EA45B+841403]
	(No symbol) [0x00007FF6C2F9345F]
	(No symbol) [0x00007FF6C2F8F4D4]
	(No symbol) [0x00007FF6C2F8F66D]
	(No symbol) [0x00007FF6C2F7EBC9]
	BaseThreadInitThunk [0x00007FFBF3FC259D+29]
	RtlUserThreadStart [0x00007FFBF5E4AF38+40]



-----

-----
Error occurred at: 2024-11-06 22:44:18.417968
Error: Error: 'NoneType' object has no attribute 'is_displayed'
Traceback: Traceback (most recent call last):
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\base\base_class.py", line 39, in scroll_to_element
    element =  WebDriverWait(self.driver, 20).until(EC.visibility_of_element_located((By.XPATH, locator)))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\.venv\Lib\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
            ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\.venv\Lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 156, in _predicate
    return _element_if_visible(driver.find_element(*locator))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\.venv\Lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 179, in _element_if_visible
    return element if element.is_displayed() == visibility else False
                      ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'is_displayed'


-----

-----
Test: tests/test_scenario_2.py::test_scenario_2 (call)
Start time: 2024-11-06 22:44:22.678642
Start name method: Scenario 2 start


-----
Test: tests/test_scenario_3.py::test_scenario_3 (call)
Start time: 2024-11-06 22:44:29.608520
Start name method: Scenario 3 start


-----
Error occurred at: 2024-11-06 22:44:36.384849
Error: Expected URL 'https://sbis.ru/download?tab=plugin&innerTab=default', but got 'https://sbis.ru/download'
Traceback: Traceback (most recent call last):
  File "C:\Users\ajnaf\PycharmProjects\tensor_test\base\base_class.py", line 90, in assert_url
    assert get_url == result, f"Expected URL '{result}', but got '{get_url}'"
           ^^^^^^^^^^^^^^^^^
AssertionError: Expected URL 'https://sbis.ru/download?tab=plugin&innerTab=default', but got 'https://sbis.ru/download'


-----
